installing/installing_aws/installing-aws-wavelength-zone.adoc

```diff
@@ -0,0 +1,214 @@
+// CloudFormation template for the VPC
+include::modules/installation-cloudformation-vpc-localzone.adoc[leveloffset=+2]
+
+// Creating a VPC Carrier Gateway
+include::modules/installation-creating-aws-vpc-carrier-gw.adoc[leveloffset=+2]
+
+// CloudFormation template for the VPC Carrier Gateway
+include::modules/installation-cloudformation-vpc-carrier-gw.adoc[leveloffset=+2]
+
+// Creating subnets in Wavelength Zones
+include::modules/installation-creating-aws-vpc-subnets.adoc[leveloffset=+2]
+
+// CloudFormation template for the subnet that uses AWS Wavelength Zones
+include::modules/installation-cloudformation-subnet-localzone.adoc[leveloffset=+2]
+
+// Modifying an installation configuration file to use AWS Wavelength Zones subnets
+include::modules/installing-aws-edge-zones-custom-vpc-config.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about viewing the CloudFormation stacks that you created, see link:https://console.aws.amazon.com/cloudformation[AWS CloudFormation console].
+* For more information about AWS profile and credential configuration, see link:https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html[Configuration and credential file settings] in the AWS documentation.
+
+.Next steps
+* xref:../../installing/installing_aws/installing-aws-wavelength-zone.adoc#installation-launching-installer_installing-aws-wavelength-zone[Deploying the cluster]
+
+// Creating edge worker nodes in public subnets (optional)
+include::modules/machineset-aws-edge-zone-public.adoc[leveloffset=+1]
+
+//TODO: Put this back if QE validates it: ASK MARCO
+//include::modules/installation-configure-proxy.adoc[leveloffset=+2]
+
+//TODO: Verify removal due to automation. ASK MARCO
+//include::modules/installation-localzone-generate-k8s-manifest.adoc[leveloffset=+2]
+
+// Deploying the cluster
+include::modules/installation-launching-installer.adoc[leveloffset=+1]
+
+.Next steps
+
+//* xref:../../post_installation_configuration/cluster-tasks.adoc#installation-extend-edge-nodes-aws-local-zones_post-install-cluster-tasks[Creating user workloads in AWS {zone-type}]
+
+== Verifying the deployed cluster's status
+
+Verify that your {product-title} has successfully deployed on an AWS Wavelength Zones environment.
+
+// Logging in to the cluster by using the CLI
+include::modules/cli-logging-in-kubeadmin.adoc[leveloffset=+2]
+
+// Logging in to the cluster by using the web console
+include::modules/logging-in-by-using-the-web-console.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about accessing and understanding the {product-title} web console, see xref:../../web_console/web-console.adoc#web-console[Accessing the web console] for more details about accessing and understanding the {product-title} web console.
+
+// Verifying nodes that were created with edge compute pool
+include::modules/machine-edge-pool-review-nodes.adoc[leveloffset=+2]
```

Comment:
The examples are not reflecting Wavelength zones.

It seems there are fragments missing from my original PR in that part.
https://github.com/openshift/openshift-docs/pull/69128/files#diff-15313606d3878b5defdb7eef95dc6380644736181ffaa4c3ae3f0d06cfe16b35L20-R71

I can't see the examples for WL, only Local Zones in the WL book:

https://69537--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-wavelength-zone#machine-edge-pool-review-nodes_installing-aws-wavelength-zone

![Screenshot from 2024-01-19 14-59-20](https://github.com/openshift/openshift-docs/assets/3216894/410eaf7d-6f24-4b3d-a50d-f185f1e522b8)

Reply:
Thanks. Updated. Sorry for missing those.